usaspending_api/api_contracts/contracts/v2/search/spending_by_category/recipient.md

```diff
@@ -92,8 +94,9 @@ This endpoint returns a list of the top results of Recipients sorted by the tota
     The id is the database key.
 + `name` (required, string, nullable)
 + `code` (required, string, nullable)
-    `code` is a user-displayable code (such as a program activity or NAICS code, but **not** a database ID). When no such code is relevant, return a `null`.
+    `code` is a user-displayable code (such as a program activity or DUNS code, but **not** a database ID). When no such code is relevant, return a `null`.
```

Comment:
Ayub, I have a minor change for this one. I think it would be better to call out that this is actually a `DUNS` code rather than as an example of a potential type of code it could be. Could be something like this:
> The Recipient's DUNS code. If it doesn't have a DUNS code, this field will be `null`